service/service.py

```diff
@@ -0,0 +1,44 @@
+"""
+Customers Service
+
+This service will create, add, update, list, read, and delete customer profiles.
+"""
+
+import os
+import sys
+import logging
+from flask import Flask, jsonify, request, url_for, make_response, abort
+from flask_api import status  # HTTP Status Codes
+
+# For this example we'll use SQLAlchemy, a popular ORM that supports a
+# variety of backends including SQLite, MySQL, and PostgreSQL
+from flask_sqlalchemy import SQLAlchemy
+from service.models import YourResourceModel, DataValidationError
+
+# Import Flask application
+from . import app
+
+######################################################################
+# GET INDEX
+######################################################################
+@app.route("/")
+def index():
+    """ Root URL response """
+    return (
+        jsonify(
```

Comment:
Replace jsonify(.....) with a simple statement. We will put this back later after we build the lists route. For now I think it is trying to set the "/" route to the lists route which we have not yet built. We will add this back in with the lists story.

Reply:
Here is what my code looks like right now:
@app.route("/")
def index():
    """ Root URL response """
    return (
         "Reminder: return some useful information in json format about the service here", 
        status.HTTP_200_OK,
    )